edgelet/edgelet-http-workload/Cargo.toml

```
@@ -19,7 +19,7 @@ percent-encoding = "2"
 regex = "1"
 serde = "1"
 serde_json = "1"
-tokio = "1"
+tokio = { version = "1", features = ["parking_lot"] }
```

```suggestion
tokio = { version = "1", features = ["parking_lot", "sync"] }
```
`parking_lot` does not expose `tokio::sync`.

---

`tokio = { version = "1", features = ["macros", "rt"] }` should be added to the development dependencies as well.